Tests/syntax_test_logical_operators.pb

```diff
@@ -1,26 +1,52 @@
-﻿; SYNTAX TEST "Packages/PureBasic/PureBasic.sublime-syntax"
+﻿;; SYNTAX TEST "Packages/PureBasic/PureBasic.sublime-syntax"
+
+; For info, see "Variables and Types":
+; https://www.com/documentation/reference/variables.html
 
 EnableExplicit
 
 Global Left = #True
 Global Right = #False
 
-If Left And Right
-;       ^^^ keyword.logical.purebasic
-    Debug #True
-EndIf
-
-If Left Or Right
-;       ^^ keyword.logical.purebasic
-    Debug #True
-EndIf
-
-If Not Left And Right
-;  ^^^ keyword.logical.purebasic
-    Debug #True
-EndIf
-
-If Left XOr Right
-;       ^^^ keyword.logical.purebasic
-    Debug #True
-EndIf
+If Left And Right : Debug #True : EndIf
+;;      ^^^   keyword.operator.word
+
+If Left Or Right : Debug #True : EndIf
+;;      ^^   keyword.operator.word
+
+If Not Left And Right : Debug #True : EndIf
+;; ^^^            keyword.operator.word
+;;          ^^^   keyword.operator.word
+
+If Left XOr Right : Debug #True : EndIf
+;;      ^^^   keyword.operator.word
+
+; Test for case-sensitivity
```

Comment:
Don't these have the exact same capitalization? Did you mean to write them in lower case?

Reply:
Yes I did intend to, but must have forgot.

Reply:
Fixed